test/controllers/BaseNavigationControllerSpec.scala

```diff
@@ -231,7 +231,7 @@ class BaseNavigationControllerSpec extends SpecBase with GuiceOneAppPerSuite wit
     }
   }
 
-  "validateIndex" - {
+  "validateIndexAsync" - {
```

Comment:
Should this be done in the BaseItemNavControllerSpec instead? This remains unchanged?